soccer_vision_3d_msgs/scripts/visualizer.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+
+import rclpy
+import time
+from rclpy.node import Node
+from rclpy.duration import Duration
+
+from geometry_msgs.msg import Vector3
+from std_msgs.msg import ColorRGBA
+from visualization_msgs.msg import Marker
+from soccer_vision_3d_msgs.msg import BallArray, FieldBoundary, GoalpostArray, RobotArray
+from soccer_vision_attribute_msgs.msg import Confidence, Robot as RobotAttribute
+
+
+class SoccerVision3DMarkers(Node):
+    """This node provides RViz markers coresponding to the recognized objects."""
+
+    def __init__(self):
+        super().__init__('SVM2DRviz')
```

Comment:
This does not match the class name anymore. Also, nobody will understand this abbreviation, why not spell it out?